src/main/java/org/kuali/kfs/module/ar/ArPropertyConstants.java

```diff
@@ -257,6 +257,15 @@ public static class CashControlDocumentFields {
 
     public static class CashControlDetailFields {
         public static final String CASH_CONTROL_DETAILS_TAB = "document.cashControlDetails";
+
+        /* CU Customization: Backport FINP-10040 These constants can be removed 2023-09-20 financials patch. */
```

Comment:
the FederalFormReportFields sub class in this file has gotten out of sync with base code.  I wonder if it would be good to catch that up now.
@cah292 do you have an opinion on this?

Reply:
@jhulslander The reason why the `FederalFormReportFields` nested class is out-of-sync is because we previously backported the FINP-7147 changes into this file. The sections regarding the FINP-7147 backport should be kept as-is until we upgrade to the related KualiCo patch version (2023-06-28).